src/ert/config/gen_kw_config.py

```diff
@@ -196,8 +179,7 @@ def from_config_list(cls, gen_kw: list[str | dict[str, str]]) -> Self:
             )
         return cls(
             name=gen_kw_key,
-            forward_init=forward_init,
-            forward_init_file=init_file,
+            forward_init=False,
```

Comment:
Do we maybe want to remove this argument? (or is it in use for fields/surfaces?)

Reply:
forward_init is defined in `ParameterConfig` and used in both fields and surfaces. So it needs to be set to false :/